README.md

```diff
@@ -32,6 +32,8 @@ To get the Retail Demo Store running in your own AWS account, follow these instr
 
 ### AWS Deployment Options
 
+***IMPORTANT NOTE:** Please use only letters in Cloud Formation Stack name.*
```

Comment:
Just curious, is there a specific reason for using just letters?

Reply:
unfortunately most of our services have a different naming rules. Problem with numbers comes from Lex (the support only letters) and as we define an IAM roles in CFN with resource based scope we have to define Lex name during deploy stage.

Reply:
Got it, thanks!